src/riverroute/RtmIO.F90

```diff
@@ -1928,14 +1931,6 @@ subroutine ncd_getiodesc(ncid, ndims, dims, dimids, xtype, iodnum)
 
     ! Initialize the decomposition for PIO
 
-    if (xtype == pio_double ) then
-       basetype = PIO_DOUBLE
-    else if (xtype == pio_real) then
-       basetype  = PIO_DOUBLE
```

Comment:
@jedwards4b can you comment on why basetype is set to PIO_DOUBLE even when input xtype=PIO_REAL? Thanks